src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -85,16 +85,8 @@ private static void InitColorTable()
                     // Retrieving getter of the category property of the UserPreferenceChangingEventArgs.
                     Type argsType = Type.GetType("Microsoft.Win32.UserPreferenceChangingEventArgs, System, Version = 4.0.0.0, Culture = neutral, PublicKeyToken = b77a5c561934e089", throwOnError: false);
```

Comment:
Why is this going through the System shim?

Reply:
I think line 61 explains that ("We use the desktop identity for SystemEvents since it is stable and will remain stable for compatibility whereas the core identity could change.").

Reply:
If we're worried about startup time, though, doesn't going through the System shim force the additional DLL to be resolved, loaded, etc.?  And "whereas the core identity could change", I'm not understanding that... we build this assembly for netcoreapp, can't we use the .NET Core assembly identity when targeting netcoreapp, and if this type ever moves, we need to update this accordingly? (That's the case for all such reflection we do in core to workaround layering.)

Reply:
I guess I'll @Anipik answer that then since I don't know more about this than what's documented in the code. Would we change this for `Microsoft.Win32.SystemEvents`?

I was going after JITtting and this avoids jitting two methods (the helper, and eventually the lambda within the helper since we don't have R2R code for that either). Avoiding type loads for the two `Func` instances is a bonus. I guess the System shim could be another bonus.

I'm hoping we can get this in today since I'm not sure this would meet the bar tomorrow. WinForms startup perf is a death by a thousand cuts and this is just one of the many cuts.

Reply:
My question/comment shouldn't block this PR.

Reply:
> "We use the desktop identity for SystemEvents since it is stable and will remain stable for compatibility whereas the core identity could change"

It might have been true before we had everything figured, but I do not think this statement is valid anymore. The .NET Core is not going to change. It would be a breaking change, just like it would be a breaking change to change System.dll. I think it is pretty safe to change this to load the actual .NET Core assembly, without going through the shim.